.github/workflows/ci.yaml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version:
+        - '2.7'
+        activerecord:
+        - "52"
+        - "60"
+    services:
+      postgres:
+        image: postgres:13
```

Comment:
I was going to switch this over to manageiq/postgresql:10 for consistency, but for some reason I was getting unhealthy containers.  For now, this is fine.